### ISSUE TITLE
ML forecasting notebook 1 chapter 8 breaks

### ISSUE THREAD

**Issue by giambascientist86** [PacktPublishing/Modern-Time-Series-Forecasting-with-Python#46]
get_X_y breaks in this notebook beacuse of this error; passing a set instead of list ; I have modified the code by passing a list(set---) but it breaks..
<img width="854" alt="error ML forecasting " src="https://github.com/user-attachments/assets/67e3745e-e322-469b-8743-9aeb3095bfb7">

**Comment by giambascientist86** [comment]
solved.

**Comment by sumantabasak** [comment]
Same problem I'm getting. How did you solve? Can you share the resolved code? @giambascientist86

**Comment by manujosephv** [comment]
doesnt the below solve the issue:

``` python
df.loc[:, list(set(feature_list + self.index_cols))]
```

**Comment by sumantabasak** [comment]
> doesnt the below solve the issue:
> 
> ```python
> df.loc[:, list(set(feature_list + self.index_cols))]
> ```

Yes, this solves the problem. Thanks @manujosephv. Can you please update it in github repo as well? It'll help, as these functions are valuable, and anyone can use directly. Thanks.
